### PR TITLE
Issue #7 change blogpost.html

### LINKED ISSUE #7: In blogpost.html for each template, add in the Jekyll tags needed for most data items.
For each template, provide `blogpost.html` mostly finished, so add in the templating logic to populate the following items:

* Blog title
* Placeholder image
* Category for loop

Status:

- [ ] Done for dopetrope
- [ ] Done for verti
- [ ] Done for hyperspace

### PR DESCRIPTION
**Pull request by SuzeShardlow**
Closes #7

### CODE REVIEW

File: html-templates/dopetrope/blogpost.html [hunk 1]
```diff
@@ -40,12 +40,12 @@ <h1><a href="homepage.html">TODO Site Name</a></h1>
 								<!-- Content -->
 									<section>
 										<header class="major">
-                                            <h2>TODO Blog Title</h2>
+                                            <h2>{{post.title}}</h2>
```
Comment (simonprickett) [inline html-templates/dopetrope/blogpost.html]: This should be `page`.

File: html-templates/dopetrope/blogpost.html [hunk 2]
```diff
@@ -40,12 +40,12 @@ <h1><a href="homepage.html">TODO Site Name</a></h1>
...
                                         </header>
 										<div class="row">
 											<div class="col-12">
-												<span class="image fit"><img src="https://via.placeholder.com/1600x1200" alt="Main Image" /></span>
-												<em>Dec 01 2019 in CATEGORY, CATEGORY, CATEGORY</em>
+												<span class="image fit"><img src="{{site.baseurl}}/assets/images/{{post.main_image}}" alt="Main Image" /></span>
+												<em>Dec 01 2019 in {{% for category in post.categories %}}{{category}} {{% endfor %}}</em>
```
Comment (simonprickett) [inline html-templates/dopetrope/blogpost.html]: This should be `page`.

File: html-templates/dopetrope/blogpost.html [hunk 3]
```diff
@@ -40,12 +40,12 @@ <h1><a href="homepage.html">TODO Site Name</a></h1>
...
 									<section>
 										<header class="major">
-                                            <h2>TODO Blog Title</h2>
+                                            <h2>{{post.title}}</h2>
                                         </header>
 										<div class="row">
 											<div class="col-12">
-												<span class="image fit"><img src="https://via.placeholder.com/1600x1200" alt="Main Image" /></span>
-												<em>Dec 01 2019 in CATEGORY, CATEGORY, CATEGORY</em>
+												<span class="image fit"><img src="{{site.baseurl}}/assets/images/{{post.main_image}}" alt="Main Image" /></span>
```
Comment (simonprickett) [inline html-templates/dopetrope/blogpost.html]: This should be `page`.

**Comment by SuzeShardlow** [comment]
Changes made.

**Pull request merged**